compiler/rustc_middle/src/ty/query/on_disk_cache.rs

```diff
@@ -936,7 +936,7 @@ impl<'a, 'tcx> Decodable<CacheDecoder<'a, 'tcx>> for &'tcx [Span] {
 
 //- ENCODING -------------------------------------------------------------------
 
-/// An encoder that can write the incr. comp. cache.
+/// An encoder that can write to the incremental compilation cache.
```

Comment:
Is this correct? Does the CacheEncoder write *to* an existing cache, or does it create a new one each time? If it creates a new cache, I think we'll want to leave out the word "to".

Reply:
I think the cache is represented by `OnDiskCache`, so this is fine.